### PR TITLE
sepolia: add geth exporter

### PR DESCRIPTION
**Pull request by siddarthkay**
This PR adds geth exporter so that I can observer syncing stats while doing a full sync of the nodes.

status: changes applied

### CODE REVIEW

File: ansible/sepolia.yml [hunk 1]
```diff
@@ -17,10 +17,10 @@
...
   roles:
     - { role: update-netplan,           tags: [ update-netplan ], when: ansible_system == 'Linux' }
-    - { role: infra-role-open-ports,    tags: [ open-ports ]        }
-    - { role: infra-role-smart-metrics, tags: [ smart-metrics ]     }
-    - { role: redirect-ports,           tags: [ redirect-ports ]    }
-    - { role: nimbus-era-files,         tags: [ nimbus-era-files ]  }
+    - { role: infra-role-open-ports,    tags: [ open-ports ]                                      }
+    - { role: infra-role-smart-metrics, tags: [ smart-metrics ]                                   }
+    - { role: redirect-ports,           tags: [ redirect-ports ]                                  }
+    - { role: nimbus-era-files,          tags: [ nimbus-era-files ]                                 }
```
Comment (jakubgs) [inline ansible/sepolia.yml:23]: Wrong indent.

File: ansible/sepolia.yml [hunk 2]
```diff
@@ -52,3 +52,11 @@
       loop_control:
         loop_var: node
         index_var: idx
+
+    - include_role:
+        name: infra-role-geth-exporter
+        apply:
+          tags: always
+      tags: [ infra-role-geth-exporter ]
```
Comment (jakubgs) [inline ansible/sepolia.yml:60]: The tag is for operator convenience, it doesn't need to be that long.
```suggestion
      tags: [ geth-exporter ]
```

File: ansible/group_vars/nimbus.sepolia.yml [hunk 3]
```diff
@@ -89,7 +99,6 @@ validator_client_keymanager_token: '{{lookup("bitwarden", "nimbus/keymanager", f
 validator_client_dist_validators_enabled: '{{ node.start is defined and node.end is defined }}'
 validator_client_dist_validators_start: '{{ (not node.vc) | ternary(0, node.start) | mandatory }}'
 validator_client_dist_validators_end:   '{{ (not node.vc) | ternary(0, node.end)   | mandatory }}'
-
```
Comment (jakubgs) [inline ansible/group_vars/nimbus.sepolia.yml:92]: Why was this empty line separating different configuration removed?

File: ansible/group_vars/nimbus.sepolia.yml [hunk 4]
```diff
@@ -78,6 +81,13 @@ validator_client_build_frequency: 'daily'
 # Ports
 validator_client_metrics_port:    '{{ 8108 + idx|int + 1 }}'
 validator_client_keymanager_port: '{{ 5052 + idx|int + 1 }}'
+
+# Geth metrics¬
+geth_expo_service_name: '{{ geth_service_name }}'
+geth_expo_source_cont_name: '{{ geth_cont_name }}'
+geth_expo_source_data_path: '{{ geth_cont_vol }}/data'
+geth_expo_cont_port:  '{{ 9400 + (idx|int) + 1 }}'
```
Comment (jakubgs) [inline ansible/group_vars/nimbus.sepolia.yml:89]: You're not opening the port on the firewall for this.

**Review by jakubgs — CHANGES_REQUESTED** [review]

File: ansible/sepolia.yml (hunk 1 above)
Comment (siddarthkay) [reply ansible/sepolia.yml:23]: this is very weird, looks okay in my editor

Screenshot 2024-07-30 at 5 29 02 PM

However, looks messed up in github UI

Screenshot 2024-07-30 at 5 29 22 PM

Comment (siddarthkay) [reply ansible/sepolia.yml:23]: looks messed up in vim as well, so I shall fix
Screenshot 2024-07-30 at 5 30 46 PM

File: ansible/sepolia.yml (hunk 2 above)
Comment (siddarthkay) [reply ansible/sepolia.yml:60]: I agree, fixed.

File: ansible/group_vars/nimbus.sepolia.yml (hunk 3 above)
Comment (siddarthkay) [reply ansible/group_vars/nimbus.sepolia.yml]: my bad, added this line break back

File: ansible/group_vars/nimbus.sepolia.yml (hunk 4 above)
Comment (siddarthkay) [reply ansible/group_vars/nimbus.sepolia.yml:89]: fixed in latest commit

**Review by jakubgs — APPROVED** [review]
Looks good, thanks.

**Comment by siddarthkay** [comment]
I've applied these changes

**Pull request merged**